src/main/java/online/book/store/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,20 @@
 package online.book.store.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
use `@NotBlank` for String

```suggestion
    @NotBlank
```